Week_04/Lecture_8/assignment2/src/main/java/org/assignment1/assignment2/repository/employeeRepository.java

```diff
@@ -0,0 +1,41 @@
+package org.assignment1.assignment2.repository;
+
+import org.assignment1.assignment2.model.employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class employeeRepository {
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public List<employee> getAllEmployees() {
+        String sql = "select * from employee";
+        return jdbcTemplate.query(sql, new BeanPropertyRowMapper<>(employee.class));
+    }
+
+    public employee getEmployeeById(int id) {
+        String sql = "select * from employee where id = ?";
+        return jdbcTemplate.queryForObject(sql, new Object[]{id}, new BeanPropertyRowMapper<>(employee.class));
```

Comment:
What happen if there is no provided id in this query?

Reply:
I updated the documentations where we put the endpoint for getting employee by ID, thank you Ms for the question.